src/main/java/Duke.java

```diff
@@ -1,10 +1,172 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
+import java.util.regex.Pattern;
+import java.util.regex.Matcher;
```

Comment:
Personally, I sort the import statements by alphabetical ordering, perhaps you might want to consider doing that as well.